lib/logstash/filters/translate.rb

```diff
@@ -138,17 +138,17 @@ def load_yaml(registering=false)
     begin
       @dictionary.merge!(YAML.load_file(@dictionary_path))
     rescue Exception => e
+      msg = "#{self.class.name}: #{e.message} when loading dictionary file at #{@dictionary_path}"
       if registering
-        raise "#{self.class.name}: Bad Syntax in dictionary file #{@dictionary_path}"
+        raise RuntimeError.new(msg)
```

Comment:
this information is already included in the message coming out of the exception.